neonvm/controllers/virtualmachine_controller.go

```diff
@@ -598,6 +606,29 @@ func affinityForVirtualMachine(virtualmachine *vmv1.VirtualMachine) *corev1.Affi
 	return a
 }
 
+func notifyRunner(vm *vmv1.VirtualMachine, r resource.Quantity) error {
+	client := http.Client{Timeout: 5 * time.Second}
+
+	url := fmt.Sprintf("http://%s:%d/cpu_change", vm.Status.PodIP, vm.Spec.RunnerPort)
+
+	update := api.VCPUChange{VCPUs: r}
+
+	data, err := json.Marshal(update)
+	if err != nil {
+		return err
+	}
+
+	resp, err := client.Post(url, "application/json", bytes.NewReader(data))
+	if err != nil {
+		return err
+	}
+
+	if resp.StatusCode != 200 {
+		return fmt.Errorf("unexpected status %d", resp.StatusCode)
```

Comment:
I'd use `resp.Status` which is already a string.

do we want to do `!= 200` or just `>= 400` are there non 200 responses that aren't errors. 

do we want to do any amount of retrying? on these requests

Reply:
The retryring is already there because this code runs inside the Kubernetes reconciliation loop